ghost/core/core/server/data/schema/commands.js

```diff
@@ -156,6 +156,20 @@ async function dropColumn(tableName, column, transaction = db.knex, columnSpec =
     }
 }
 
+/**
+ * @param {string} tableName
+ * @param {string} from
+ * @param {string} to
+ * @param {import('knex').Knex.Transaction} [transaction]
+ */
+async function renameColumn(tableName, from, to, transaction = db.knex) {
```

Comment:
We need to be careful here: https://ghost.slack.com/archives/C02G9E68C/p1662985535756619 and https://github.com/TryGhost/Ghost/commit/a6991746c3aec3e925e2d50f4e88edf87c8dcc74

Reply:
Thanks for the heads up! I was planning to ask for your review anyway because of this. Is it okay if I implement this raw query approach in the helpers instead? I think we'll have to use this in the future migrations again, and probably good to have a helper for it

Reply:
I've updated the code for this change